fgread/dataset.py

```diff
@@ -20,6 +20,10 @@ def __init__(self, path: str):
             raise FileNotFoundError(self.path)
 
         self.metadata = self.read_metadata()
+        assert self.metadata["schemaVersion"] == "1.0", (
```

Comment:
Check auf hardgecodete Version? ;-) würde ich perspektivisch in ne Variable ändern. Ich mecker aber auch nur aus Böswilligkeit und Langeweile....

Reply:
Ja, weil das für den alten Reader ist und dieser nur noch exakt mit dieser Version funktioniert.
Sollte der Reader vor dem Multifile released werden würde der Alte so lange halt noch funktionieren.